### DISCUSSION TITLE
Broken images

### DISCUSSION THREAD

**Discussion by MisinformationScout** [community/community#21701] [Other Feature Feedback, Questions, & Ideas]
<p>I’m trying to upload a website but some of the images don’t load for some reason. Here’s a link to me repository if anyone can see if I’m doing something wrong. I also reuploaded everything as a folder hoping that would fix it. </p>
<p><a href="https://github.com/MisinformationScout/MisinformationScout" rel="nofollow noopener">https://github.com/MisinformationScout/MisinformationScout</a></p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Things seem to be working properly today.</p>
<p>Let us know if you have more questions or I’m misunderstanding.</p>

**Comment by whtitefall** [comment]
<p>How did you solve the problem, I got the same issue </p>
